extensions/amp-youtube/1.0/component.js

```diff
@@ -218,3 +248,7 @@ function makeMethodMessage(method) {
     })
   );
 }
+
+const Youtube = forwardRef(YoutubeWithRef);
+Youtube.displayName = 'YouTube'; // Make findable for tests.
```

Comment:
I think we should either rename the component `YouTube`/`YouTubeWithRef` or set its display name value to `'Youtube'` - that is keep these cases consistent.